framework/Distributions.py

```diff
@@ -439,6 +439,15 @@ def pdf(self,x):
     returnPdf = self._distribution.pdf(x)
     return returnPdf
 
+  def logpdf(self,x):
```

Comment:
I am having my revenge here: should it be logPdf?

Reply:
fixed.